src/metabase/models/card.clj

```diff
@@ -59,6 +59,14 @@
   (derive :hook/timestamped?)
   (derive :hook/entity-id))
 
+(defmethod mi/can-write? Card
+  ([instance]
+   (if (= (t2/select-one-fn :entity_id 'Collection :id (:collection_id instance)) (perms/default-audit-collection-entity-id))
```

Comment:
nit: I think we're supposed to be using `:models/Collection` to refer to the model in toucan 2 (see the docstring for the `Collection` var). Same with `Card` below.

Reply:
Also a nit, but I think it would be nice to have a comment here (and for the `Dashboard` implementation) explaining the extra logic. Just like `Cards in audit collection should be read only` or something